README.md

```diff
@@ -1,73 +1,101 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# HTML & CSS3 & JavaScript Course
 
-# Project Name
+If you are not familiar with linters, read [root level README](../README.md).
 
-> Description the project.
+## Set-up GitHub Actions
 
+Please do the following **steps in this order**:
 
-## Built With
```

Comment:
- [ ] This is the correct format of Readme. Kindly make sure to update your readme with the correct headings.